### ISSUE TITLE
Several issues with LeptonX beta 3 (MVC)

### ISSUE THREAD

**Issue by cbogner85** [volosoft/lepton-theme#9]
Hi,

I've intensively tested LeptonX (MVC), as I'm very excited about it. 
I encountered several issues, some have been fixed with beta.3 (e.g. the tenantbox is now showing :)), but the following issues still exist:

- Login/ Register: Page Alerts aren't displayed; if you enter wrong username/ password, no error message is displayed; there is also no error message if you register and the username already exists. 
- "Linked Accounts" user menu item is not working
- Breadcrumb clicks aren't working 
- User Toolbar has small display error, the chevron icon is missing, therefore it's quite confusing

Additionally, I have a question: Some colors are still "hand-coded" in bootstrap-[themename].css, e.g. btn-primay. If we change the colors from the variables, the buttons are still blue. Therefore, we have to replace all the colors manually using search and replace, which is not a huge problem, but means that we have to do it with every update again. Do you plan to use css variables for all classes in future?

btw, I love the new design! Great work!

PS: I'm using LeptonX Commercial.

**Comment by ismcagdas** [comment]
Thanks a lot @cbogner85 for your tests, we really appriciate it. @enisn could you create separate issues for each test on our LeptonX repo ?

**Comment by enisn** [comment]
@cbogner85 Thanks for your feedback. We'll work on these.

**Comment by cbogner85** [comment]
> * User Toolbar has small display error, the chevron icon is missing, therefore it's quite confusing

Forgot to mention: the chevron icon is only missing in mobile view.

**Comment by cbogner85** [comment]
Hi @ismcagdas  & @enisn 

I found another issue:

I localized the theme names and hided the SemiDark item. 
On desktop, it works as expected:
![desktop](https://user-images.githubusercontent.com/49239838/177990990-cb45597c-4598-49c9-bfa6-0be0322614c0.PNG)


On mobile, my localizations aren't applied and also the SemiDark item still shows.
![mobile](https://user-images.githubusercontent.com/49239838/177991001-95331b2d-1d8d-4e39-923a-afe4f8abea20.PNG)

BTW: is this repo the right place for bug reports or should we better use abp repo (or even support.apb.io?).

**Comment by ismcagdas** [comment]
Yes, you can report the problems here. We will check the issue and fix it, thanks.

**Comment by cbogner85** [comment]
Hello, 

the Page Alert issue still persists in RC.4 - do you have any plans to fix it? The issue breaks many processes, e.g. register, login, reset password,... 

I worked around by overriding necessary classes and displaying the alerts myself, however, I forgot to override reset password method, therefore users didn't receive any error when they didn't fulfill password complexity. 

I know Lepton-X is still not final, but this is a real show-stopper and should be fixed before releasing 1.0 stable.

Best regards
Claus

**Comment by ismcagdas** [comment]
Thanks a lot Claus. @enisn could you check the Page Alert issue with the latest version ? 

> Login/ Register: Page Alerts aren't displayed; if you enter wrong username/ password, no error message is displayed; there is also no error message if you register and the username already exists.

**Comment by cbogner85** [comment]
Hello,

most of the issues are fixed! 

Regarding the missing chevron icon in mobile user menu (still present), I found out that there is an unneccessary bi class which hides the fa icons (obviously copied from lepton html theme where bootstrap icons are used):

![image](https://user-images.githubusercontent.com/49239838/197549815-64961c8b-06d0-4ed3-919d-23bd2cf03489.png)

If we simply remove this class, the icons appear:

![image](https://user-images.githubusercontent.com/49239838/197550000-4357421b-2390-44f7-84aa-0c8a238f032b.png)

Maybe this could be fixed easliy :)

**Comment by ismcagdas** [comment]
@cbogner85 thanks a lot. @enisn could you create an issue on volo repository for this ?
